### ISSUE TITLE
Wrong informations when activating setMaxTrackingAge, with MonitorNotifier and RangeNotifier

### ISSUE THREAD

**Issue by meme-ppm** [AltBeacon/android-beacon-library#342]
My application receives wrong informations from **MonitorNotifier** and **RangeNotifier** when using **RangedBeacon.setMaxTrackinAge()**

My application starts monitoring a iBeacon Region based on the UUID, the Major, the Minor (named **Beacon Region**) using **BeaconManager.startMonitoringBeaconsInRegion()**

My Application next starts ranging on a iBeacon Region based on  the UUID (named **UUID Region**) to detect all my beacons around. 

My application is notified about on exit from the **Beacon Region**, but the method **didRangeBeaconsInRegion** continues to return the beacon for several seconds (while I receive an exit from the beacon region).

I hope my case is clear.

**Comment by davidgyoung** [comment]
`RangedBeacon.setMaxTrackingAge()` only controls the period of time ranged beacons will continue to be returned after the scanning service stops detecting them.  It has no affect on when monitored regions trigger exits.  It is set to 5 seconds by default.

Monitored regions are exited whenever a scan period finishes and the `BeaconManager.setRegionExitPeriod()` has passed since the last detection.  By default, this is 10 seconds, but you can customize it.

Using the defaults, the library will stop sending ranging updates five seconds after a beacon was last seen, and then send a region exit 10 seconds after it was last seen.  You are welcome to change these two settings to meet your needs, but the `BeaconManager.setRegionExitPeriod()` should generally be the same or longer than the `RangedBeacon.setMaxTrackingAge()`, otherwise you will see issues like you describe.

To summarize: this is working as designed and should be expected behavior, given that `RangedBeacon.setMaxTrackingAge()` is a method on the RangedBeacon class, implying the setting is specific to ranging.
